src/transformers/models/convnext/image_processing_convnext_fast.py

```diff
@@ -62,11 +60,16 @@ class ConvNextFastImageProcessorPreprocessKwargs(DefaultFastImageProcessorPrepro
     crop_pct: Optional[float]
 
 
-@add_start_docstrings(
+@customize_docstrings(
     r"Constructs a fast ConvNeXT image processor.",
-    BASE_IMAGE_PROCESSOR_FAST_DOCSTRING,
     """
-        crop_pct (`float`, *optional*):
+        size (`Dict[str, int]` *optional*, defaults to `{"shortest_edge": 384}`):
+                Resolution of the output image after `resize` is applied. If `size["shortest_edge"]` >= 384, the image is
+                resized to `(size["shortest_edge"], size["shortest_edge"])`. Otherwise, the smaller edge of the image will
+                be matched to `int(size["shortest_edge"]/crop_pct)`, after which the image is cropped to
+                `(size["shortest_edge"], size["shortest_edge"])`. Only has an effect if `do_resize` is set to `True`. Can
+                be overriden by `size` in the `preprocess` method.
+        crop_pct (`float`, *optional*, defaults to `224/256`):
             Percentage of the image to crop. Only has an effect if size < 384. Can be
             overridden by `crop_pct` in the`preprocess` method.
     """,
```

Comment:
Maybe it would be better to use smth like `@with_default_docstring(DEFAULT_FAST_PREPROCESS_DOCSTRING)` decorator, which will

1. Select args from DEFAULT_DOCSTRING according to the method signature
2. Update the args docstrings according to the actually defined docstrings (similar to what we have right now for the model)


Exampe:

```
@with_default_docstring(DEFAULT_FAST_PREPROCESS_DOCSTRING)
def preprocess(...):
    """
    Args:
        size: 
            <new size definition here>
        crop_pct: 
            <crop pct definition here>
   """
```
WDYT?